app/assets/javascripts/discourse/app/controllers/composer.js

```diff
@@ -722,6 +722,13 @@ export default Controller.extend({
       this.cancelComposer();
     },
 
+    saveDraft() {
+      this._saveDraft();
```

Comment:
Since we are keeping `saveDraftAndClose` which then calls `onSaveDraft`, we can just call this function from `onSaveDraft` since it does the same 4 things. So this:

```javascript
onSaveDraft: () => {
  this._saveDraft();
  this.model.clearState();
  this.close();
  this.appEvents.trigger("composer:cancelled");
  resolve();
},
```

Becomes this:

```javascript
onSaveDraft: () => {
  this.saveDraft();
  resolve();
},
```

Reply:
How does the composer.js file have access to the method in discard-draft.js?

Reply:
I get an error trying this 🙁

Reply:
So, I pushed the change but can't get it to work.